etlman/projects/admin.py

```diff
@@ -16,21 +16,19 @@ class StepInline(admin.TabularInline):
 
 
 class ProjectAdmin(admin.ModelAdmin):
-    inlines = [CollaboratorInline, DataInterfaceInLine]
```

Comment:
I think you can revert the edits to `ProjectAdmin` and `DataInterfaceAdmin` now that `project` is back.